HostLibraryTests/KernelArguments_test.cpp

```diff
@@ -50,15 +50,12 @@ TEST(KernelArguments, Simple)
         float  z;
         char   t;
 
+        char padd[3]; 
+
         size_t k;
 
     } ref;
 
-    // Padding bytes would be left uninitialized in the struct but will be
-    // zero-filled by the KernelArguments class.  Set them to zero to prevent a
-    // test failure.
-    memset(&ref, 0, sizeof(ref));
```

Comment:
Using memset() on struct which contains a floating point number.

Reply:
I think I can cast `&ref` to a `void*` in the `memset` call to avoid the warning. This shouldn't be a problem with the potential invalid floating point value because they are overwritten later anyway.